src/utils/__tests__/test-geoJSON-leaflet.js

```diff
@@ -0,0 +1,105 @@
+import {geoJSONToLeafletLayers, 
+        layersToGeoJSON, 
+        layersToGeoJSONMultipolygon} from '../geoJSON-leaflet.js';
+import {fakeLeafletLayer1, 
+        fakeLeafletLayer2, 
+        polygon1, 
+        polygon2} from '../__test_data__/sample-leaflet-layers.js'
+
+describe('geoJSONToLeafletLayers', function() {});
+
+describe('layersToGeoJSON', function() {
+    const expectedSingle = {
+        "geometry": {
+            "coordinates": polygon1,
+            "type": "Polygon",
+        },
+        "properties": {
+            "source": "PCIC Climate Explorer"
+        },
+        "type": "Feature"
+    };
+    const expectedGeometry = {
+        "geometry": {
+            "geometries": [
+                {
+                    "coordinates": polygon1,
+                    "type": "Polygon"
+                },
+                {
+                    "coordinates": polygon2,
+                    "type": "Polygon"
+                }
+            ],
+            "type": "GeometryCollection"
+        },
+        "properties": { "source": "PCIC Climate Explorer"},
+        "type": "Feature"
+    };
+    
+    const expectedFeature = {
+        "features": [
+            {
+                "geometry": {
+                    "coordinates": polygon1,
+                    "type": "Polygon",
+                },
+                "properties": { "source": "PCIC Climate Explorer"},
+                "type": "Feature"
+            },
+            {
+                "geometry": {
+                    "coordinates": polygon2, 
+                    "type": "Polygon"
+                },
+                "properties": {},
+                "type": "Feature",
+            },
+        ],
+        "properties": {
+            "source": "PCIC Climate Explorer"
+        },
+        "type": "FeatureCollection"
+    };
```

Comment:
There's a lot of repetition here, but it's probably not worth the effort to DRY it up. Everything is *nearly* the same as everything else but not quite.